### DISCUSSION TITLE
¿Tendrá acceso a Internet para documentación?

### DISCUSSION THREAD

**Discussion by ChunyLabs** [community/community#56365] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hola! he estado probando el GitHub Copilot Chat y le he hecho una pregunta que me dejó con dudas.

![image](https://github.com/community/community/assets/119270803/8e0ae128-77b7-478d-a404-2296aeb16111)

¿Este modelo tendrá algún acceso a internet o los datos serán actualizados con el tiempo? ya que la versión 12 fue lanzada hace ya algún tiempo.

**Comment by Akash1134** [comment]
Hi @M8-Babbage ,

Thanks for participating! We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
